l10n_fr_pos_caisse_ap_ip/static/src/js/payment_caisse_ap_ip.js

```diff
@@ -0,0 +1,105 @@
+/*
+    Copyright 2023 Akretion France (http://www.akretion.com/)
+    @author: Alexis de Lattre <alexis.delattre@akretion.com>
```

Comment:
could you add camptocamp as financial contributor please ? thanks @alexis-via

Reply:
The place should not be in the header of the file. (but can be set in a CREDITS.rst file, in the readme section).